templates/web/browseS3.ftl

```diff
@@ -161,11 +161,15 @@
       });
     </script>
 
+    <#include "/static-assets/app/pages/legacy.html">
     <script type="text/javascript">
-        $(function() {
+      document.addEventListener("DOMLegacyReady", () => {
+        CrafterCMSNext.system.getStore().subscribe(() => {
```

Comment:
Let's add a `store$` fn to `craftercms` global that reexports store.ts getStore so we can use that and not use `CrafterCMSNext` in any of these updates. Update all these type of usages. Explicitly, use `window.craftercms`.
```suggestion
      document.addEventListener("DOMLegacyReady", () => {
        window.craftercms.store$().subscribe(() => {
```

Now, that being said, wherever possible, don't even invoke store$() if not necessary. It might be enough for the custom event listener. Check how it worked before, I don't think all these waited for or needed store readyness.

Reply:
Without the store being ready the api calls fail due to lack of token (unauthenticated). Though I'm still not sure how it worked before